### PR TITLE
Add step for bootstrap or sync and section on multiple VFNs to valida…

### PR DESCRIPTION
**Pull request by clay-aptos**
…tor updates doc

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ Since you are already running [a validator node and a validator fullnode](node-r
 
 This page explains how to make this swap, which largely amounts to switching out files and configuration settings between the two nodes. For a community-provided version of this document for Docker setup, see [Failover and migrate Validator Nodes for less downtime](https://forum.aptoslabs.com/t/failover-and-migrate-validator-nodes-for-less-downtime/144846).
 
+## Secure
```
Comment (JoshLind) [inline developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:14]: I'm not sure if "Secure" is the right title 🤔 I'd probably have a new section at the bottom of this document, where the title is something like "Running multiple Validator Fullnodes".

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ Since you are already running [a validator node and a validator fullnode](node-r
...
+
+Do not run multiple VFNs with the same [network identity](../../identity-and-configuration.md) and connect them to the validator using the `vfn` network, as this may cause issues with node metrics and telemetry. Nevertheless, we realize you may need access to REST APIs for building without any rate limits.
```
Comment (JoshLind) [inline developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md:16]: nit: I'd maybe rephrase to something like:
```
If you wish to run multiple VFNs, e.g., to expose REST APIs for building products without any rate
limits, follow the instructions below. Note that it is not currently recommended to run multiple VFNs
with the same [network identity](../../identity-and-configuration.md) and connect them to the
validator using the `vfn` network, as this may cause issues with node metrics and telemetry.
```

**Review by JoshLind — APPROVED** [review]
LGTM. Thanks @clay-aptos 😄

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md]: Thanks! I did this and also rewrote the intro a bit to match the flow.

File: developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md]: Done!
